tests/cpp-tests/Resources/Shaders/example_outline.fsh

```diff
@@ -25,7 +25,7 @@ void main()
     
     accum *= u_threshold;
     accum.rgb =  u_outlineColor * accum.a;
-    accum.a = 1.0;
+    //accum.a = 1.0; why set to 1.0 ?
```

Comment:
Could you please modify it in another pull request.
We can discuss it there.
It is better to do one thing for a pull request.